distributed/scheduler.py

```diff
@@ -7130,7 +7130,7 @@ async def feed(
         Caution: this runs arbitrary Python code on the scheduler.  This should
         eventually be phased out.  It is mostly used by diagnostics.
         """
-        if not dask.config.get("distributed.scheduler.pickle"):
+        if not dask.config.get("distributed.scheduler.pickle"):  # pragma: no cover
```

Comment:
This is an actual feature. We should test it

```suggestion
        if not dask.config.get("distributed.scheduler.pickle"):
```